### ISSUE TITLE
Null safety issue in datadog-plugin-kafkajs consumer tracing

### ISSUE THREAD

**Issue by NiXXeD** [DataDog/dd-trace-js#3063]
**Expected behaviour**
When instrumenting kafkajs consumers, it should not crash the consumer.

**Actual behaviour**
It is possible to consume a message from kafkajs that has headers that contain a null value. When this happens, the instrumentation code [here](https://github.com/DataDog/dd-trace-js/blob/master/packages/datadog-plugin-kafkajs/src/consumer.js#L33) fails with `TypeError: Cannot read property 'toString' of null`. This causes the consumer to crash and not restart.

**Steps to reproduce**
Produce a message with a header value that is null.

ex:
```json
{
  "headers": {
    "valid": "value",
    "invalid": null
  }
}
```

**Comment by kuntza01** [comment]
This issue appears to have been corrected by #3098 , which was just released in v3.20.0.

**Comment by tlhunter** [comment]
Looks like this issue was solved so I'll close it for now. If this isn't the case please ping me and reopen!
